src/test/rpc_tests.cpp

```diff
@@ -335,7 +335,7 @@ BOOST_AUTO_TEST_CASE(rpc_insightexplorer)
     CheckRPCThrows("getspentinfo {\"a\":1}",
         "Error: getspentinfo is disabled. "
         "Run './zcash-cli help getspentinfo' for instructions on how to enable this feature.");
-    CheckRPCThrows("getblockdeltas \"a\"",
+    CheckRPCThrows("getblockdeltas a",
```

Comment:
This seems like a change that should be documented in the release notes. It's possible that some other changes in this PR should also be documented there.